core/mono-samples/Android/AndroidSampleApp.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices;
+
+// it's not part of the BCL but runtime needs it for native-to-managed callbacks in AOT
+// To be replaced with NativeCallableAttribute
+public class MonoPInvokeCallbackAttribute : Attribute
```

Comment:
We probably can get rid of it using Function Pointers actually but I'm not sure, last time I checked - callbacks still needed it.

Reply:
And it's not needed for JIT (which is the only option for Android atm)

Reply:
It works after removing this block as well as the attributes. Does this still qualify as showing how the embedded mono runtime allows for C# and C to call each other with just the function pointers and `Dllimport`?